flask_httpauth.py

```diff
@@ -62,12 +63,12 @@ def authenticate_header(self):
 
     def get_auth(self):
         auth = request.authorization
-        if auth is None and 'Authorization' in request.headers:
+        if auth is None and self.header in request.headers:
```

Comment:
This would give you the wrong result if the client sends `Authorization` header as well as a custom header, I think, because you only look at `self.header` when `self.authorization` is None. If you are passing a custom header then the existence of `Authorization` in the request should not matter.

Reply:
Good point, didn't think of it! should be fixed now.